src/mcnp/mcnp_funcs.cpp

```diff
@@ -198,37 +201,37 @@ void write_cell_cards(std::ostringstream& lcadfile, const char* mcnp_version_maj
       // that material numbers are assigned
       mat_num = DMD->volume_material_data_eh[entity];
       // if we cant make an int from the mat_num
-      if (mat_num.find("Graveyard") == std::string::npos &&
-          mat_num.find("Vacuum") == std::string::npos) {
+      if (mat_num.find(graveyard_str) == std::string::npos &&
+          mat_num.find(vacuum_str) == std::string::npos) {
         if (!DMD->try_to_make_int(mat_num)) {
-          std::cout << "Failed to cast material number to an integer"  << std::endl;
-          std::cout << "volume with ID " << cellid << " has material assignment" << std::endl;
-          std::cout << mat_num << " which appears to be a name rather than an integer" << std::endl;
-          std::cout << "Did you forget to run uwuw_preproc?" << std::endl;
+          std::cerr << "Failed to cast material number to an integer"  << std::endl;
+          std::cerr << "volume with ID " << cellid << " has material assignment" << std::endl;
+          std::cerr << mat_num << " which appears to be a name rather than an integer" << std::endl;
+          std::cerr << "Did you forget to run uwuw_preproc?" << std::endl;
           exit(EXIT_FAILURE);
         }
       }
 
       density = DMD->volume_density_data_eh[entity];
       // if we have a vacuum problem
-      if (mat_num == "Graveyard" || mat_num == "Vacuum") {
+      if (mat_num == graveyard_str || mat_num == vacuum_str) {
         mat_num = "0";
         density = "";
       }
     } else {
       std::string mat_name = DMD->volume_material_property_data_eh[entity];
       // if we not vacuum or graveyard
-      if (mat_name.find("Vacuum") == std::string::npos && mat_name.find("Graveyard") == std::string::npos) {
+      if (mat_name.find(vacuum_str) == std::string::npos && mat_name.find(graveyard_str) == std::string::npos) {
         if (workflow_data->material_library.count(mat_name) == 0) {
-          std::cout << "Material with name " << mat_name <<  " not found " << std::endl;
-          std::cout << "In the material library" << std::endl;
+          std::cerr << "Material with name " << mat_name <<  " not found " << std::endl;
+          std::cerr << "In the material library" << std::endl;
           exit(EXIT_FAILURE);
         }
 
         pyne::Material material = workflow_data->material_library[mat_name];
         int matnumber = material.metadata["mat_number"].asInt();
-        mat_num = _to_string(matnumber);
-        density = "-" + _to_string(material.density);
+        mat_num = std::to_string(matnumber);
+        density = "-" + std::to_string(material.density);
```

Comment:
should we also use `_to_string()` here?

Reply:
yes- I haven't gone back through to untangle all the `to_string()`/`_to_string()` uses yet